wolfcrypt/src/ecc.c

```diff
@@ -5795,12 +5795,19 @@ int wc_ecc_make_key(WC_RNG* rng, int keysize, ecc_key* key)
 WOLFSSL_ABI
 int wc_ecc_init_ex(ecc_key* key, void* heap, int devId)
 {
-    int ret = 0;
+    int ret      = 0;
+    int isPkcs11 = 0;
 
     if (key == NULL) {
         return BAD_FUNC_ARG;
     }
 
+#if defined(WOLF_PRIVATE_KEY_ID) || defined(WOLFSSL_ASYNC_CRYPT)
+    if (key->isPkcs11) {
+        isPkcs11 = 1;
```

Comment:
Why not skip this and only check for key->isPkcs11 below? I don't know about you, but I don't like ifdefs, IMO the less there are, the better :).

Reply:
Nevermind, the key is memset to 0 below.